ext/node/ops/sqlite/database.rs

```diff
@@ -9,42 +9,145 @@ use std::ffi::CString;
 use std::ptr::null;
 use std::rc::Rc;
 
+use deno_core::cppgc;
 use deno_core::op2;
-use deno_core::serde_v8;
 use deno_core::v8;
+use deno_core::v8_static_strings;
 use deno_core::GarbageCollected;
 use deno_core::OpState;
 use deno_permissions::PermissionsContainer;
 use rusqlite::ffi as libsqlite3_sys;
+use rusqlite::ffi::SQLITE_DBCONFIG_DQS_DDL;
+use rusqlite::ffi::SQLITE_DBCONFIG_DQS_DML;
 use rusqlite::limits::Limit;
-use serde::Deserialize;
 
 use super::session::SessionOptions;
+use super::statement::check_error_code2;
+use super::validators;
 use super::Session;
 use super::SqliteError;
 use super::StatementSync;
-use crate::ops::sqlite::SqliteResultExt;
+
 const SQLITE_DBCONFIG_ENABLE_LOAD_EXTENSION: i32 = 1005;
 const SQLITE_DBCONFIG_ENABLE_ATTACH_WRITE: i32 = 1021;
 
-#[derive(Deserialize)]
-#[serde(rename_all = "camelCase")]
 struct DatabaseSyncOptions {
-  #[serde(default = "true_fn")]
   open: bool,
-  #[serde(default = "true_fn")]
   enable_foreign_key_constraints: bool,
   read_only: bool,
-  #[serde(default = "false_fn")]
   allow_extension: bool,
+  enable_double_quoted_string_literals: bool,
 }
 
-fn true_fn() -> bool {
-  true
-}
+impl DatabaseSyncOptions {
+  fn from_value(
```

Comment:
Is this a new method from `deno_core` to automatically deserialize instead of using `serde_v8`?

Reply:
It appears it is - I don't get any intellisense for that at all 😬 I like the idea of it instead of `serde_v8` but I somehow lost where we do that. Can you please elaborate?

Reply:
Updated `DatabaseSyncOptions` to `FromV8`. `ApplyChangesetOptions` cannot use `FromV8` because of lifetime issues with holding Local references.